<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TicTacToe</string>
  </property>
  <widget class="QPushButton" name="resetBTN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="scorePlayer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="scoreBot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="botVSbotRB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
